pkgs/tools/nix/nixos-option/.clang-tidy

```diff
@@ -0,0 +1,22 @@
+Checks:
+- bugprone-*
+# don't find them too problematic
+- -bugprone-easily-swappable-parameters
+- performance-*
+- modernize-*
```

Comment:
No sure that https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-trailing-return-type.html is necesssary. The diff is too noisy for very little benefit. It's ok either way

Reply:
removed.